README.md

```diff
@@ -2,12 +2,11 @@
 These are the three Python files I use that allows Twitch Chat or Youtube Chat to control your keyboard or mouse to play a game. You are welcome to use or adapt this code for your own content.
 
 To run the code you will need to install Python 3.9.  
-Additionally, you will need to install the following python modules using Pip:  
-python -m pip install keyboard  
-python -m pip install pydirectinput  
-python -m pip install pyautogui  
-python -m pip install pynput  
-python -m pip install requests  
+Additionally, you will need to install the required python modules from the `requirements.txt` file using:
+```bash
+pip3 install -r requirements.txt
```

Comment:
It's better to keep the "python -m pip" command as it is since the "pip3" command can be different depending on your operating system and version.

Reply:
That's fair, however if it's going to cause issues I believe the person running the command would know which one they need to use.